angr/analyses/cfg/cfg_emulated.py

```diff
@@ -2418,7 +2419,7 @@ def _try_resolving_indirect_jumps(self, sim_successors, cfg_node, func_addr, suc
 
         return successors
 
-    def _backward_slice_indirect(self, cfgnode, sim_successors, current_function_addr):
+    def _backward_slice_indirect(self, cfgnode: object, sim_successors: object, current_function_addr: object) -> object:
```

Comment:
You may want to use `typing.Any` here rather than `object`

Reply:
that was an accident on my part adding the typing for this function, I'll remove it.